src/backup/after_backup_action_handler.py

```diff
@@ -30,8 +38,14 @@ def __init__(self, request=None, response=None):
     def post(self, **_):
         request_body_json = JsonRequestHelper.parse_request_body(self.request.body)
         self.__validate(request_body_json)
-        self.__process(request_body_json)
-        self._finish_with_success()
+
+        try:
+            self.__process(request_body_json)
+            self._finish_with_success()
+        except CopyJobError, e:
```

Comment:
I used to think that returning 200 here was intended, because afterBackupActionHandler is not responsible for retrying copyJobs etc. 
We were calling ErrorReporting to know that backups finished in not expected way. We want to know about that situation. 

I case of this errors, we can't do anything with then, even if we retry whole task again.